browser.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
+/* globals alert */
+
+const semver = require('semver')
+
+const getInstalledVersion = require('./lib/check-node')
+const loadVersions = require('./lib/load.js')
+// const install = require('./lib/install')
+// for testing purpose
```

Comment:
@delvedor Sorry – sure! I mean, do you want the testing code here included when the PR is merged?